include/wx/generic/splitter.h

```diff
@@ -323,9 +323,18 @@ class WXDLLIMPEXP_CORE wxSplitterEvent : public wxNotifyEvent
     {
         SetEventObject(splitter);
         if (splitter) m_id = splitter->GetId();
+
+        if (type == wxEVT_SPLITTER_SASH_POS_CHANGED
+                || type == wxEVT_SPLITTER_SASH_POS_CHANGING
+                || type == wxEVT_SPLITTER_SASH_POS_RESIZE)
+        {
+            m_data.sash.oldSize = 0;
+            m_data.sash.newSize = 0;
```

Comment:
I'd initialize the fields always, without checking for the event type, just to avoid having some junk in them unexpectedly later if more events using them are added.